conan/tools/google/bazeldeps.py

```diff
@@ -557,14 +634,27 @@ def __init__(self, conanfile):
 
     def generate(self):
         """
-        Generates all the targets <DEP>/BUILD.bazel files and the dependencies.bzl one in the
-        build folder. It's important to highlight that the dependencies.bzl file should be loaded
-        by your WORKSPACE Bazel file:
+        Generates all the targets <DEP>/BUILD.bazel files, a dependencies.bzl (for bazel<7), a
+        conan_deps_repo_rules.bzl and a conan_deps_module_extension.bzl file (for bazel>=7.1) one in the
+        build folder.
+
+        In case of bazel < 7, it's important to highlight that the dependencies.bzl file should
+        be loaded by your WORKSPACE Bazel file:
 
         .. code-block:: python
 
             load("@//[BUILD_FOLDER]:dependencies.bzl", "load_conan_dependencies")
             load_conan_dependencies()
+
+        In case of bazel >= 7.1, the conan_deps_module_extension.bzl file should be loaded by your
+        Module.bazel file, e.g. like this:
+
+        .. code-block:: python
```

Comment:
I added an example in the documentation repository

Reply:
Besides that, you can run `conan new bazel_lib_7 -d name=hello -d version=1.0` to see how it works (using Conan from source and this git branch for sure).

Reply:
I mean, there is an alternative using the `include` directive, that helps to separate concerns and contention on the MODULE.bazel file, and it would be nice to see this option in docs:

MODULE.bazel:
```
include("//deps/conan:conan.MODULE.bazel")
```

deps/conan/conan.MODULE.bazel:
```
    load_conan_dependencies = use_extension("//:conan_deps_module_extension.bzl", "load_dependencies")
    use_repo(load_conan_dependencies, "zlib")
```

Reply:
@peakschris could you please add the URL to the documentation? I think it could be quite interesting what you're proposing 😁

Reply:
The link can be found here: https://bazel.build/rules/lib/globals/module#include

But somehow I can't get it running…
```
ERROR: /home/[…]MODULE.bazel:1:1: name 'include' is not defined
ERROR: Error computing the main repository mapping: error executing MODULE.bazel file for <root>

```

Reply:
It's only in 7.2.0rc1 and higher.

Reply:
Then, it is better to wait for a more stable version.